lesson 17/index.js

```diff
@@ -0,0 +1,91 @@
+"use strict"
+// task 1
+const a = 'true';
+const b = 5 > 6;
```

Comment:
Тут нужно было просто написать. По факту тот же результат, но так проще
```suggestion
const b = false;
```